ttt/lib/ttt/board.ex

```diff
@@ -1,16 +1,19 @@
 defmodule TTT.Board do
+  @x_mark "X"
```

Comment:
I have this down to hightlight.  Sloppy but I'm aware of it.  I have been rushing to get finished.